target/linux/ath79/image/generic.mk

```diff
@@ -251,6 +270,11 @@ define Device/compex_wpj531-16m
   DEVICE_VENDOR := Compex
   DEVICE_MODEL := WPJ531
   DEVICE_VARIANT := 16M
+  IMAGES += cpximg-7A03.bin cpximg-7A04.bin cpximg-7A06.bin cpximg-7A07.bin
+  IMAGE/cpximg-7A03.bin = $$(IMAGE/sysupgrade.bin) | mkmylofw 0x68a 2
+  IMAGE/cpximg-7A04.bin = $$(IMAGE/sysupgrade.bin) | mkmylofw 0x693 3
+  IMAGE/cpximg-7A06.bin = $$(IMAGE/sysupgrade.bin) | mkmylofw 0x693 3
+  IMAGE/cpximg-7A07.bin = $$(IMAGE/sysupgrade.bin) | mkmylofw 0x693 3
```

Comment:
The parameters for the last three lines are the same. Mistake?

Reply:
no. they're taken from the official images.
the images produced with these parameters are accepted by the corresponding boards.
the redundancy is a trade-off for usability and ease of integration into openwrt.
in the software i've created for flashing large amounts of various compex boards, i've simply created symlinks.
the variant, e.g. `7A07`, is not only printed on the board but also part of the serial number barcode.

Reply:
Okay, I see it the same way, just wanted to make sure.